.github/workflows/test-download-artifacts.yml

```diff
@@ -22,15 +22,15 @@ jobs:
           workflow_run_id: ${{ github.event.workflow_run.id }}
           artifacts: test.download
           # the following two lines are default values included for clarity.
-          target_dir: ${{ github.workspace }}/downloads
+          target_dir: ./downloads/
```

Comment:
nit - i am not sure if i can recall this being the relative path in the future.  Can you leave a comment that GHA use. path relative to the workspace root?

Reply:
Sure, this is true of all relative paths in the file.